docs/learning/tutorials/create-test-imodel-sample.md

```diff
@@ -0,0 +1,35 @@
+# Create a test iModel from a Bentley provided sample via the Free Trial service
+The quickest way to get access to a cloud hosted iModel is by creating an iModel seeded with Bentley provided sample data.
+
+- Go to the [registration dashboard](../../../getting-started/registration-dashboard/?tab=1&create=bentleyExample)
+- Click the "+ New iModel" button
+- Give your test iModel a name
+- Select a dataset from the dropdown.

```

Comment:
nit: mind either dropping the `.` or add it to the other bullets.  I forgot if we have a good standard in our docs or not...